### PR TITLE
etl: Add expected parameter old to etl example

### LINKED ISSUE #556: etl: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by Mofeywalker**
Fixes #556

### CODE REVIEW

File: exercises/etl/etl.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-def transform():
+def transform(old):
```
Comment (m-a-ge) [inline exercises/etl/etl.py]: `legacy_data` will look better. 
What do you think?

**Comment by m-a-ge** [comment]
Merged, thanks, @Mofeywalker 

It's a best practice to squash your commits https://github.com/exercism/docs/blob/master/contributing/git-basics.md#squashing. It isn't a big deal since I can do it myself, but it would be awesome to just click on 'Merge pull request' button 😃

Please also note that there are some special ['closing issue' keywords](https://help.github.com/articles/closing-issues-using-keywords/) that you can put in description of your PR and Github will take care of closing the corresponding issue automatically after merge. 
It makes our work much easier. I hope this hint will help you to make your future PRs a bit better

Just for future reference a guide of how to write good commit messages - https://chris.beams.io/posts/git-commit/#seven-rules

**Comment by Mofeywalker** [comment]
Thanks for the hints! I haven't used Github much other than for my own school related projects. So I'm still learning some best practices, especially in collaboration. :)

**Comment by m-a-ge** [comment]
@Mofeywalker it's ok, that is what Hacktoberfest is for

**Pull request merged**